.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -0,0 +1,48 @@
+name: Feature request
+description: Suggest an idea for this project
+labels: [enhancement]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this issue! :hugs:
+        
+        However note the following before opening an issue:
+        * IF YOU DON'T FILL IN THE TEMPLATE PROPERLY, YOUR ISSUE IS LIABLE TO BE CLOSED.
+        * If you feel tired/lazy right now, open your issue some other time. We'll wait.
+
+  - type: checkboxes
+    id: checklist
+    attributes:
+      label: "Checklist"
+      options:
+        - label: "I checked, but didn't find any duplicates ([open](https://github.com/TeamNewPipe/NewPipe/issues) OR [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed)) of this issue in the repo"
```

You should avoid these caps and the brackets.

```suggestion
        - label: "I made sure that there are *no existing issues*, neither [open](https://github.com/TeamNewPipe/NewPipe/issues) nor [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed), which I could contribute my information to."
```